yarn-project/txe/src/txe_service/txe_service.ts

```diff
@@ -791,4 +791,14 @@ export class TXEService {
     }
     return toForeignCallResult([toArray(witness)]);
   }
+
+  emitUnencryptedLog(_contractAddress: ForeignCallSingle, _message: ForeignCallArray, _counter: ForeignCallSingle) {
```

Comment:
Had to add this in order to not make TXE throw an error.

Reply:
Let's do an issue as this is a bit of a footgun - I predict future readers looking at this and being angry at us.